lib/lists/list_entitylistview.dart

```diff
@@ -47,7 +51,12 @@ class EntityListView extends ConsumerWidget {
                                       entityDoc.data()!['entitiesAddress']]
                               : 'Location: undefined'),
                           isThreeLine: true,
+                          tileColor: ref.watch(selectedEntity).toString() ==
```

Comment:
It's better approach to put this condition in selected attribute instead of using it like this, for example:
selected: condition ? true : false,
selectedTileColor: color ,

Reply:
yes, I see your solution, and it's better than mine.

Reply:
And I think that maybe we also need to add the highlight when we click the list tile in the left part. If you agree with me, I can close this PR, and finish the left part, then create a new PR.

Reply:
![Screenshot 2023-01-20 at 11 30 13](https://user-images.githubusercontent.com/94722949/213592363-98e7e635-a26c-41d3-91cf-78f4a8f1e49a.png)

Reply:
Yes, your right

Reply:
You can commit changed to code to this PR as well, that's alright